### PR TITLE
chore: move to sn-gh-workflows

### PR DESCRIPTION
**Pull request by niekvanstaveren**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,183 +1,15 @@
...
-    name: Save yarn cache
-    paths:
-      - ~/.cache/yarn
-    key: yarn-packages-{{ .Branch }}-{{ checksum "yarn.lock" }}
-  - &filter_master
-    branches:
-      only: master
-  - &attach
-    at: ~/project
+version: 2.1
```
Comment (cbt1) [inline .circleci/config.yml:1]: Why an empty workflow?

**Comment by cbt1** [comment]
You should make the `Build` action required once merge

**Review by cbt1 — APPROVED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (niekvanstaveren) [reply .circleci/config.yml:1]: Because the circleci flow is required. We can switch that to the Build action once this is merged :)

**Pull request merged**